author/views.py

```diff
@@ -1,3 +1,13 @@
-from django.shortcuts import render
+from django.views import generic
+from .models import Author
 
-# Create your views here.
+class BloggerDetailView(generic.DetailView):
+    model = Author
+    template_name = "author/blogger_detail.html"
+    context_object_name = "author"
+    
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        author = self.get_object()
+        context['auth_posts'] = author.blogs.all().order_by('-created_at')
```

Comment:
please remove this .all() because you are using .order_by('-created_at') it will give us all of the shorted data.
.all() will execute query and  .order_by('-created_at') will also exclute same query and then short the data.

so query take time when we have a plenty of data